firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerOnboardingViewController.swift

```diff
@@ -56,62 +62,91 @@ class PasswordManagerOnboardingViewController: SettingsViewController {
+            ])
+            scrollView.addSubview(contentView)
+            NSLayoutConstraint.activate([
+                contentView.widthAnchor.constraint(equalTo: scrollView.frameLayoutGuide.widthAnchor),
+                contentView.topAnchor.constraint(equalTo: scrollView.contentLayoutGuide.topAnchor),
+                contentView.bottomAnchor.constraint(greaterThanOrEqualTo: scrollView.bottomAnchor),
+                contentView.heightAnchor.constraint(greaterThanOrEqualTo: scrollView.heightAnchor)
+            ])
+            contentView.addSubviews(onboardingMessageLabel, learnMoreButton, continueButton)
+            configureContentViewChildConstraints()
+    }
 
-        self.title = .Settings.Passwords.Title
+    @objc
+    func learnMoreButtonTapped(_ sender: UIButton) {
+        let viewController = SettingsContentViewController(windowUUID: windowUUID)
+        viewController.url = SupportUtils.URLForTopic("set-passcode-and-touch-id-firefox")
+        navigationController?.pushViewController(viewController, animated: true)
+    }
 
-        self.view.addSubviews(onboardingMessageLabel, learnMoreButton, continueButton)
+    @objc
+    func proceedButtonTapped(_ sender: UIButton) {
+        continueFromOnboarding()
+    }
 
+    private func configureContentViewChildConstraints() {
         NSLayoutConstraint.activate(
             [
                 onboardingMessageLabel.leadingAnchor.constraint(
-                    equalTo: self.view.safeAreaLayoutGuide.leadingAnchor,
+                    equalTo: contentView.safeAreaLayoutGuide.leadingAnchor,
                     constant: UX.standardSpacing
                 ),
                 onboardingMessageLabel.trailingAnchor.constraint(
-                    equalTo: self.view.safeAreaLayoutGuide.trailingAnchor,
+                    equalTo: contentView.safeAreaLayoutGuide.trailingAnchor,
                     constant: -UX.standardSpacing
                 ),
-                onboardingMessageLabel.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
-                onboardingMessageLabel.centerYAnchor.constraint(equalTo: self.view.centerYAnchor),
-
-                learnMoreButton.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
-                learnMoreButton.topAnchor.constraint(
-                    equalTo: onboardingMessageLabel.safeAreaLayoutGuide.bottomAnchor,
+                onboardingMessageLabel.centerXAnchor.constraint(
+                    equalTo: contentView.safeAreaLayoutGuide.centerXAnchor
+                ),
+                onboardingMessageLabel.topAnchor.constraint(
+                    equalTo: contentView.safeAreaLayoutGuide.topAnchor,
                     constant: UX.standardSpacing
                 ),
-
-                continueButton.bottomAnchor.constraint(
-                    equalTo: self.view.layoutMarginsGuide.bottomAnchor,
-                    constant: -UX.standardSpacing
+                learnMoreButton.centerXAnchor.constraint(
```

Comment:
The `learnMoreButton` is missing constraints for leading and trailing as it wouldn't be constraint into all direction otherwise. So the button label is too long the text could be partially outside of the view.

Please add:
```                
                learnMoreButton.leadingAnchor.constraint(
                    greaterThanOrEqualTo: contentView.leadingAnchor,
                    constant: UX.buttonHorizontalPadding
                ),
                learnMoreButton.trailingAnchor.constraint(
                    lessThanOrEqualTo: contentView.trailingAnchor,
                    constant: -UX.buttonHorizontalPadding
                ),
```